recommandation.go

```diff
@@ -0,0 +1,9 @@
+package recombee
+
+type Recommendations struct {
```

Comment:
missing godoc. What is the purpose for this new strucutre. How should be used. Maybe add example test? If not create issue in this project so in godoc is an example of usage of `Recommendations`

Reply:
Issue created: https://github.com/moderntv/recombee/issues/7#issue-2009718738